### ISSUE TITLE
[NFR] Documentar su clase, hints para los IDE's

### ISSUE THREAD

**Issue by gtux** [culqi/culqi-php#16]
Los que usamos su librería oficial estaríamos agradecidos si ayudan al IDE para que genere automáticamente el descubrimiento de las clases y métodos internos, una forma podría ser agregar lo que el estándar PHPDoc menciona, y así tendríamos:

```php

/**
 * Clase Culqi
 *
 * @property-read Cargos Cargos
 * @property-read Suscripciones Suscripciones
 * @property-read Devoluciones Devoluciones
 * @property-read Planes Planes
 *
 * @author Grover Manuel Campos Ancajima <grover@gttech.pe>
 * @version Dec 14, 2016
 * @copyright GT Tech E.I.R.L. <http://www.gttech.pe> (cambiarlo por su marca claro)
 */
class Culqi 
{
....
}

```

**Comment by marti1125** [comment]
@gtux te refieres a esto https://www.phpdoc.org/
https://www.phpdoc.org/templates

**Comment by gtux** [comment]
@marti1125 Sí, pero específicamente al [@property-read](https://www.phpdoc.org/docs/latest/references/phpdoc/tags/property-read.html), que es útil sobretodo en clases que tienen propiedades dinámicas, en este caso Culqi no tiene como atributos las clases Cargos o Suscripciones, sino que se establecen en el constructor, por lo que los IDE's no tienen conocimiento de esta variable y menos de su tipo, así que no te lo proponen al momento de escribir el código. 

Al documentar la clase con @property-read el IDE asume que de alguna manera existe ese atributo y es de tal tipo así que te lo ofrece en el autocompletado de código.

**Comment by marti1125** [comment]
@gtux listo lo reviso mañana 👍 algun otro feedback que tuvieras?

**Comment by marti1125** [comment]
@gtux hola!! aun lo sigues necesitando.... =/ aun el la lib para la version de la nueva api se sigue modificando

**Comment by gtux** [comment]
No, no esperé a que hicieran la actualización, yo mismo lo arreglé en mi copia.

**Comment by marti1125** [comment]
@gtux la version 1.3.* de culqi-php ya tiene docs
